concept/read_api_test.go

```diff
@@ -177,7 +184,7 @@ func TestUnhappyGTG(t *testing.T) {
 	csAPI := NewReadAPI(testClient, s.URL, apiKey, batchSize)
 
 	err := csAPI.GTG()
-	assert.EqualError(t, err, "concept search API returned a non-200 HTTP status code: 503")
+	assert.Error(t, err)
```

Comment:
We should be able to use "assert.True(t, errors.Is(err, ErrUnexpectedResponse))"